AnnoDesigner/model/LayoutObject.cs

```diff
@@ -0,0 +1,391 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Media;
+using AnnoDesigner.Core.Models;
+
+namespace AnnoDesigner.model
+{
+    public class LayoutObject
```

Comment:
Might be worth adding some comments here about what this class does - caches and optimises etc.

Reply:
Godd catch.